src/presentation/ui/component/structural/tabHeader.templ

```diff
@@ -0,0 +1,28 @@
+package componentStructural
+
+type TabHeaderItem struct {
+	Label             string
+	ComponentValue    string
+	AdditionalClasses string
+}
+
+templ TabHeader(
+	tabHeaderItems []TabHeaderItem,
+	onClick string,
+) {
+	<!-- TabHeader -->
+	<ul class="flex text-sm w-fit font-medium text-center text-os-500 uppercase rounded-t-lg bg-os-800 pt-3.5 px-4">
+		for _, item := range tabHeaderItems {
+			<li class="text-xl">
+				<button
+					type="button"
+					class={ "inline-block py-4 px-5 rounded-t-md bg-inherit hover:bg-os-300" + item.AdditionalClasses }
+					@click={ onClick + "('" + item.ComponentValue + "')" }
+				>
+					<i class="ph-duotone ph-database align-middle text-2xl mr-3"></i>
+					{ item.Label }
+				</button>
+			</li>
+		}
+	</ul>
+}
```

Comment:
Look at that. A gigantic piece of code turned into 28 lines including the struct. Zen programmer.

Reply:
😎